README.md

```diff
@@ -38,7 +38,7 @@ brew install ergo
 
 ### Linux
 ```
-curl -s https://raw.githubusercontent.com/cristianoliveira/ergo/master/install.sh | sh
+curl -s https://raw.githubusercontent.com/cristianoliveira/ergo/master/install.sh | bash
```

Comment:
Hey @ebastos Is it really necessary?

I think the `sh` is most common than `bash` for unix in general isn't?
https://en.wikipedia.org/wiki/Bourne_shell

Reply:
Correct, except for Ubuntu (which may be a large fraction of the public):

```shell
$ cat /etc/issue.net 
Ubuntu 17.04
$ ls -ld /bin/sh
lrwxrwxrwx 1 root root 4 Jun  6 17:53 /bin/sh -> dash
```
Which causes the installer to fail:

```shell
$ curl -s https://raw.githubusercontent.com/cristianoliveira/ergo/master/install.sh | sh
sh: 5: declare: not found
sh: 6: Syntax error: "(" unexpected
```

A possible way to get the install script more POSIX would be to get rid of some bashisms. We can do this using case instead of the current associative array.

Another way would be just using x86_64 instead of linux-amd64 and make it simpler.

Let me know your thoughts and I can work something out.

Reply:
I would like to be as POSIX as possible then we can ensure of most shells will work :)

Reply:
I didn't notice here cause I have changed my shell to `zsh` hehehe ouch

Reply:
I opened this issue https://github.com/cristianoliveira/ergo/issues/67 for this problem :)